code/Model/SiteTree.php

```diff
@@ -277,6 +277,13 @@ class SiteTree extends DataObject implements PermissionProvider, i18nEntityProvi
      */
     private static $icon = null;
 
+    /**
+     * Class attached to page icons in the CMS page tree. Also supports font-icon set.
+     * @config
+     * @var string
+     */
+    private static $icon_class = 'font-icon-page';
```

Comment:
Can we document this somewhere?

Reply:
Yeah sure thing, I'll create a PR later today 🙂